core/src/main/kotlin/org/evomaster/core/problem/api/param/Param.kt

```diff
@@ -9,16 +9,26 @@ abstract class Param(
         val genes : MutableList<Gene>
 ) : StructuralElement(genes){
 
+    /**
+     * Contains the description of the parameter.
+     */
+    var description: String? = null
+        set(value) {
+            if (!value.isNullOrEmpty() && field.isNullOrEmpty()) {
```

Comment:
i guess if a `field` is already set we should throw an IllegalStateException, stating can only be set once? also, add comment here to specify why we do this. also, if this is just on Param, we could force it to be set on constructor time (ie Param is not so widely used as Gene, requiring lot of changes). ah, never mind, there is issue with copy() :) still clarify it in comments